filament/backend/include/backend/DriverEnums.h

```diff
@@ -1219,6 +1222,20 @@ struct StencilState {
     uint8_t padding = 0;
 };
 
+struct PushConstant {
+    using Variant = std::variant<int, float, bool>;
+    char const* name = nullptr;
+    Variant value;
+};
+
+using PushConstantArray = std::array<PushConstant, MAX_PUSH_CONSTANT_COUNT>;
+
+struct PushConstantStruct {
```

Comment:
This shouldn't be needed in this header. Probably just in Program.

Reply:
removed.